localization/english/dvg_flavor_genoa_l_english.yml

```diff
@@ -0,0 +1,43 @@
+﻿l_english:
+#Journal Entries
+ #je_caladeran_war_of_independence_spain:0 "The Andino Revolt"
+ #je_caladeran_war_of_independence_spain_reason:0 "The Andino Revolt must be quelled!"
+ 
+ #Events
+ #dvg_spain.1.t:0 "The Andino Revolt!"
+ #dvg_spain.1.d:0 "Treachery!"
+ #dvg_spain.1.f:0 "The Independence of Gran Colombia has sent ripples throughout Arcadia and Ameriga, and these ripples have finally reached New Toledo. They have #abandoned the Spanish Crown and seek independence."
+ #dvg_spain.1.a:0 "¡Viva España!"
+
+ #Modifiers
+ #dvg_giant_big_spain:0 "Gran Imperio Español"
+ 
+ #Names
+ ig_bolognese_school:0 "The Bolognese School"
+ ig_bolognese_school_desc:0 ""
+ ig_genoese_patricians: 0 "Genoese Patricians"
+ ig_genoese_patricians_desc:0 ""
+ Lodovico: "Lodovico"
+ della_Rovere: "della Rovere"
+ Gastone: "Gastone"
+ Bonaparte: "Bonaparte"0
```

Comment:
Unexpected localization token '0' at line 23 and column 23 in localization/english/dvg_flavor_genoa_l_english.yml